src/assets/images/mentor_statue.svg

```diff
@@ -1,4 +1,4 @@
-<svg width="500" height="273" viewBox="25 0 262 273" fill="none" xmlns="http://www.w3.org/2000/svg">
+<svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 5.85 262 267.15">
```

Comment:
Sorry I didn't catch this earlier @RandyHuynh5815 , the statue bases are now not centered on the pedestal (looks very awkward on mobile). I think we can expand viewbox width to 300 for the mentor statue and 320 for the volunteer statue and it'll still be small enough to not affect spacing.

Reply:
@RandyHuynh5815, why did the SVG files double in size?

Reply:
I have modified the actual width of the images rather than the view box width by mistake. This issue should be resolved now.